proj/src/main/webapp/static/chat.html

```diff
@@ -146,27 +146,41 @@ <h1 class="display-4 card-header" id="big-title"> Chat Room </h1>
                 <!-- chat display-->
                 <div class="card-body" id="message-list">
                     <!-- template for later use -->
```

Comment:
I think you can remove this line now, right?

Reply:
yup it's fixed now